cmd/notification.go

```diff
@@ -1168,6 +1168,17 @@ func (args eventArgs) ToEvent() event.Event {
 func sendEvent(args eventArgs) {
 
 	// remove sensitive encryption entries in metadata.
+	switch {
+	case crypto.IsEncrypted(args.Object.UserDefined):
+		if totalObjectSize, err := args.Object.DecryptedSize(); err == nil {
+			args.Object.Size = totalObjectSize
+		}
+	case args.Object.IsCompressed():
+		if totalObjectSize := args.Object.GetActualSize(); totalObjectSize >= 0 {
```

Comment:
This type of code is not useful @poornas 

```
args.Object.Size = args.Object.GetActualSize()
```

is enough

Reply:
done